components/tools/OmeroPy/src/omero/plugins/sessions.py

```diff
@@ -247,6 +247,17 @@ def _configure(self, parser):
         for x in (file, key, logout, keepalive, list, clear, group):
             self._configure_dir(x)
 
+        open = parser.add(sub, self.open, "Create a session for "
+                                          "the given user and group")
+        self.add_user_and_group_arguments(open)
+        open.add_argument("--timeout", nargs="?", type=int, default=0,
+                          help="Timeout in seconds (optional; default: "
+                               "maximum possible)")
```

Comment:
Is it actually creating sessions that last forever? I wonder if it should default instead to whatever the usual is.

Reply:
If one creates a session with no timeout specified, it creates a session with timeout `0` and and idle timeout of `10 min` whatever that means...